lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -533,9 +628,22 @@ class TraceProcessor {
     // Find the inspected frame
     const mainFrameIds = this.findMainFrameIds(keyEvents);
 
+    const frames = keyEvents
+      .filter(evt => evt.name === 'FrameCommittedInBrowser')
+      .map(evt => evt.args.data)
+      .filter(/** @return {data is {frame: string, url: string}} */ data => {
+        return Boolean(data && data.frame && data.url);
```

Comment:
just pointing out you're using `!!(...)` and `Boolean(...)` for the same thing in this PR. is there one you'd prefer?

Reply:
Whoops, I think I'm going to stick with `Boolean(...)`

In my defense, this was some code I moved up from below.

Reply:
I am so thrilled to have you on the team @adamraine :) ⚽ 🏈 ⚾ 🎾  🏆 🥇 

Just don't tell @paulirish 😉

Reply:
I guess if the decision is such a big deal I would have put a little more thought into it.

Reply:
nah you did good